### PR TITLE
ISPN-14635 LREM command

### PR DESCRIPTION
**Pull request by karesti**
https://issues.redhat.com/browse/ISPN-14635

### CODE REVIEW

**Review by jabolina — COMMENTED** [review]
Only the command arity.

**Review by wburns — APPROVED** [review]

File: multimap/src/main/java/org/infinispan/multimap/impl/ListBucket.java [hunk 1]
```diff
@@ -250,6 +250,29 @@ public ListBucket<V> insert(boolean before, V pivot, V element) {
...
+      long maxRemovalsCount = count == 0 ?  values.size(): Math.abs(count);
+      long removedElements = 0;
+      while(ite.hasNext() && maxRemovalsCount > 0) {
+         V next = ite.next();
+         if (Objects.deepEquals(next, element)) {
+            maxRemovalsCount--;
+            removedElements++;
+            ite.remove();
+         }
+      }
```
Comment (rigazilla) [inline multimap/src/main/java/org/infinispan/multimap/impl/ListBucket.java:271]: can be optimized, something like this
```suggestion
      while(ite.hasNext()) {
         V next = ite.next();
         if (Objects.deepEquals(next, element)) {
            ite.remove();
            if (++removedElements==maxRemovalsCount) {
              break;
            }
         }
```

File: server/resp/src/main/java/org/infinispan/server/resp/commands/list/LREM.java [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+ * When key does not exist, the command will always return 0.
+ *
+ * @since 15.0
+ */
+public class LREM extends RespCommand implements Resp3Command {
+
+   public static final String COUNT = "COUNT";
+
+   public LREM() {
+      super(5, 1, 1, 1);
```
[the remark this answers is not in the document]
Comment (karesti) [reply server/resp/src/main/java/org/infinispan/server/resp/commands/list/LREM.java:36]: thanks

**Comment by karesti** [comment]
@jabolina changes reported

**Review by jabolina — APPROVED** [review]

**Comment by jabolina** [comment]
Integrated into main, thanks @karesti!

**Pull request merged**